lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
figured it out.  the References.repair_all was calling find_in_batches directly, so it didn't matter when I commented out find_each.

The reason Philipp was seeing it fail was that it was actually run on a relation object, not on the record object. 

Will push out my changes in a new branch.